app/views/jobs/_system_status.html.erb

```diff
@@ -1,20 +1,53 @@
+<%- cluster_info = get_cluster_info(jobs) -%>
+
 <div class='container border py-1 rounded-1'>
     <h2><%= cluster.capitalize() %> Cluster</h2>
-    <table class='table'>
-        <tr>
-            <th></th>
-            <th>Running</th>
-            <th>Queued</th>
-        </tr>
-        <tr>
-            <th>CPU</th>
-            <td><%= jobs.count { |job| job.status == 'running' } %></td>
-            <td><%= jobs.count { |job| job.status == 'queued' } %></td>
-        </tr>
-        <tr>
-            <th>GPU</th>
-            <td><%= jobs.count { |job| job.status == 'running' && job.native[:gres].include?("gpu") } %></td>
-            <td><%= jobs.count { |job| job.status == 'queued' && job.native[:gres].include?("gpu") } %></td>
-        </tr>
-    </table>
+    <div class="row justify-content-center">
+        <div class="col">
+            <div class="barchart">
+                <div class="bcBar bar-<%= cluster_info[:gpu_using_pct] %> bg-primary">
+                    <p class="barValue">
+                        <%= cluster_info[:gpu_using] %>
+                    </p>
+                    <p class="barLabel">
+                        GPUs in use
+                    </p>
+                </div>
+                <div class="bcBar bar-<%= cluster_info[:gpu_requesting_pct] %> bg-primary">
+                    <p class="barValue">
+                        <%= cluster_info[:gpu_requesting] %>
+                    </p>
+                    <p class="barLabel">
+                        GPUs requested
```

Comment:
```suggestion
                        GPU jobs queued
```

When I read requested - I questioned whether it meant `queued` or `queued + running`. Looking at the code it's actually queued, so let's call it that.

Also - to be crystal clear - it's GPU jobs that are pending, not the GPUs themselves by the looks of the calculation.

Reply:
Happy to keep reporting jobs for now and defer #25, but whatever we show it's got to be labeled appropriately.